hack/start.sh

```diff
@@ -18,15 +32,54 @@ EOF
     exit
 }
 
-dry_run=false
+get_image() {
+    component=$1
+    eval echo $IMAGE_FORMAT
+}
+
+cleanup(){
+    local RETURN_CODE=$?
+
+    set +e
+
+    # Save manifests for debugging if ARTIFACT_DIR is set.
+    if [ -n "$ARTIFACT_DIR" ]; then
+        mkdir -p $ARTIFACT_DIR/manifest
+        cp $MANIFEST/* $ARTIFACT_DIR/manifest/
+    fi
+
+    if [ -n "$MANIFEST" ]; then
+        rm -rf $MANIFEST
+    fi
+    exit $RETURN_CODE
+}
+
+
+DRYRUN=false
+REPO_ROOT="$(dirname $0)/.."
+OCP_VERSION=${OCP_VERSION:-4.5}
+YAML2JSON=$REPO_ROOT/hack/yaml2json.py
+IMAGE_FORMAT=${IMAGE_FORMAT:-""}
+MANIFEST=$(mktemp -d)
+trap cleanup exit
+
+# Find the latest OCP version. It's the greatest 4.x directory in /manifests dir
+OCP_VERSION=$( ls $REPO_ROOT/manifests | sort | grep "^4" | tail -n 1 )
```

Comment:
This might break in future. I am working on a follow up that renames the channel to "4.5" to "preview" and then directory will not be available.  We should extract this from CSV itself.

Reply:
not a blocker though, I can fix this in my PR.